### ISSUE TITLE
custom train/val split

### ISSUE THREAD

**Issue by johleh** [kujason/avod#115]
Dear @kujason , @melfm 

i think creating a good custom 85/15 split to train the model for the  test server submission is not trivial,
but more complex models should benefit from more data.
How do you think about publishing the split you used for your AVOD-FPN submission ? 

to even the odds and create a new standard split for test results,like the 50/50 train/val split that is now used in most papers and allows the comparison of validation results.

in any case, thanks a lot for the paper, code and explanations.

**Comment by linchunmian** [comment]
@johleh  Hi, the problem I confused is why does test data split? when we get test data from official website, it is without label information and we cannot conduct the training and validation process on test data?
I am so confused about this problem, and could you please give me an explanation?
Thanks in advance!

**Comment by kujason** [comment]
You can generate your own custom split by separating the drives by date and index to avoid repeating samples within the same sequences across training/validation. I used a simple image similarity metric to determine which sequences to keep in the training split.
@ChunmianLin The test data has no labels, so all training and evaluation has to be done on the labelled training data. The inference results on the test data can be submitted online where they are evaluated by the server.
